sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -169,64 +169,6 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder(conf) {
     }
   }
 
-  /**
-   * Create a table, returning a [[CreateTable]] logical plan.
-   *
-   * This is used to produce CreateTempViewUsing from CREATE TEMPORARY TABLE.
-   *
-   * TODO: Remove this. It is used because CreateTempViewUsing is not a Catalyst plan.
-   * Either move CreateTempViewUsing into catalyst as a parsed logical plan, or remove it because
-   * it is deprecated.
-   */
-  override def visitCreateTable(ctx: CreateTableContext): LogicalPlan = withOrigin(ctx) {
```

Comment:
this is not used at all?

Reply:
Several tests in ```DDLSuite``` uses this. I changed the tests from ```CREATE TEMPORARY TABLE``` to ```CREATE TEMPORARY VIEW```, so after the change this is not used at all.